designs/opal/package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "@freesewing/opal",
+  "version": "3.0.0",
+  "description": "A FreeSewing pattern that needs a description",
+  "author": "Joost De Cock <joost@joost.at> (https://github.com/joostdecock)",
+  "homepage": "https://freesewing.org/",
+  "repository": "github:freesewing/freesewing",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/freesewing/freesewing/issues"
+  },
+  "funding": {
+    "type": "individual",
+    "url": "https://freesewing.org/patrons/join"
+  },
+  "keywords": [
+    "freesewing",
+    "design",
+    "diy",
+    "fashion",
+    "made to measure",
+    "parametric design",
+    "pattern",
+    "sewing",
+    "sewing pattern"
+  ],
+  "type": "module",
+  "module": "dist/index.mjs",
+  "exports": {
+    ".": {
+      "internal": "./src/index.mjs",
+      "default": "./dist/index.mjs"
+    }
+  },
+  "scripts": {
+    "build": "node build.mjs",
+    "build:all": "yarn build",
+    "clean": "rimraf dist",
+    "mbuild": "NO_MINIFY=1 node build.mjs",
+    "symlink": "mkdir -p ./node_modules/@freesewing && cd ./node_modules/@freesewing && ln -s -f ../../../* . && cd -",
+    "test": "npx mocha tests/*.test.mjs",
+    "vbuild": "VERBOSE=1 node build.mjs",
+    "lab": "cd ../../sites/lab && yarn start",
+    "tips": "node ../../scripts/help.mjs",
+    "lint": "npx eslint 'src/**' 'tests/*.mjs'",
+    "prettier": "npx prettier --write 'src/*.mjs' 'tests/*.mjs'",
+    "testci": "NODE_OPTIONS=\"--conditions=internal\" npx mocha tests/*.test.mjs --reporter ../../tests/reporters/terse.js",
+    "wbuild": "node build.mjs",
+    "wbuild:all": "yarn wbuild"
+  },
+  "peerDependencies": {
+    "@freesewing/core": "3.0.0"
+  },
+  "dependencies": {},
+  "devDependencies": {
+    "mocha": "10.2.0",
+    "chai": "4.3.10",
+    "@freesewing/models": "3.0.0",
+    "@freesewing/plugin-timing": "3.0.0"
+  },
+  "files": [
+    "dist/*",
+    "README.md"
+  ],
+  "publishConfig": {
+    "access": "public",
+    "tag": "latest"
+  },
+  "engines": {
+    "node": "18",
```

Comment:
I needed to change this to a newer version as I'm testing with node 20. The other designs specify a range.

Reply:
Fixed.